### PR TITLE
add more projects

### PR DESCRIPTION
**Pull request by zoff99**
This change is [Reviewable](https://reviewable.io/reviews/toktok/website/81)

### CODE REVIEW

**Comment by iphydf** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[toktok/integrations.md, line 40 at r1](https://reviewable.io:443/reviews/toktok/website/81#-L3I81eRpYt_eqYq_Y4I:-L3I81eRpYt_eqYq_Y4J:b2ncbe4) ([raw file](https://github.com/toktok/website/blob/0074f2f3fef5940ed065e80bcfdafadf53708056/toktok/integrations.md#L40)):*
> ```Markdown
> *   [Skupina-robot](https://github.com/zoff99/skupina-robot)
> *   [Video streaming bot](https://github.com/TokTok/streambot-jvm)
> *   [ToxBildGruppenArbeiter](https://github.com/zoff99/ToxBildGruppenArbeiter) (Video Conferencing bot for multiple Users)
> ```

Sort alphabetically.

---

*[toktok/integrations.md, line 50 at r1](https://reviewable.io:443/reviews/toktok/website/81#-L3I83u-FYWS7hMfA71S:-L3I83u-FYWS7hMfA71T:bbngup9) ([raw file](https://github.com/toktok/website/blob/0074f2f3fef5940ed065e80bcfdafadf53708056/toktok/integrations.md#L50)):*
> ```Markdown
> *   [Go](https://github.com/xhebox/gtox) gtox
> *   [Haskell](https://github.com/TokTok/hs-toxcore-c)
> *   [pure Java](https://github.com/zoff99/java_toxclient_example/tree/master/jni-c-toxcore)
> ```

"Pure"

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/81)*

**Comment by zoff99** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[toktok/integrations.md, line 40 at r1](https://reviewable.io:443/reviews/toktok/website/81#-L3I81eRpYt_eqYq_Y4I:-L3IBsqdVFxLqRt39SLg:b-896fix) ([raw file](https://github.com/toktok/website/blob/0074f2f3fef5940ed065e80bcfdafadf53708056/toktok/integrations.md#L40)):*
<details><summary><i>Previously, iphydf wrote…</i></summary><blockquote>

Sort alphabetically.
</blockquote></details>

Done.

---

*[toktok/integrations.md, line 50 at r1](https://reviewable.io:443/reviews/toktok/website/81#-L3I83u-FYWS7hMfA71S:-L3IBteyrrXzE22C8jzX:bvjsfgv) ([raw file](https://github.com/toktok/website/blob/0074f2f3fef5940ed065e80bcfdafadf53708056/toktok/integrations.md#L50)):*
<details><summary><i>Previously, iphydf wrote…</i></summary><blockquote>

"Pure"
</blockquote></details>

not sure if thats what you meant?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/81)*

**Comment by iphydf** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

*[toktok/integrations.md, line 40 at r1](https://reviewable.io:443/reviews/toktok/website/81#-L3IOlgzQ8JVpLhnOJ4Y:-L3IOlgzQ8JVpLhnOJ4Z:be0ew61) ([raw file](https://github.com/toktok/website/blob/0074f2f3fef5940ed065e80bcfdafadf53708056/toktok/integrations.md#L40)):*
> ```Markdown
> *   [Skupina-robot](https://github.com/zoff99/skupina-robot)
> *   [Video streaming bot](https://github.com/TokTok/streambot-jvm)
> *   [ToxBildGruppenArbeiter](https://github.com/zoff99/ToxBildGruppenArbeiter) (Video Conferencing bot for multiple Users)
> ```

```
mdl -i -w -s .md-style.rb toktok
toktok/integrations.md:39: MD013 Line length
toktok/integrations.md:66: MD013 Line length
```

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/81)*

**Comment by robinlinden** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

*[toktok/integrations.md, line 50 at r1](https://reviewable.io:443/reviews/toktok/website/81#-L3I83u-FYWS7hMfA71S:-L3IY763nMefzUvTM8nu:b-946xso) ([raw file](https://github.com/toktok/website/blob/0074f2f3fef5940ed065e80bcfdafadf53708056/toktok/integrations.md#L50)):*
<details><summary><i>Previously, zoff99 (Zoff) wrote…</i></summary><blockquote>

not sure if thats what you meant?
</blockquote></details>

Why not just "Java"? Since the other binding is marked as "Java/Scala."

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/81)*

**Comment by zoff99** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[toktok/integrations.md, line 40 at r1](https://reviewable.io:443/reviews/toktok/website/81#-L3IOlgzQ8JVpLhnOJ4Y:-L3IevhWAEZ7nEET-wlJ:b-896fix) ([raw file](https://github.com/toktok/website/blob/0074f2f3fef5940ed065e80bcfdafadf53708056/toktok/integrations.md#L40)):*
<details><summary><i>Previously, iphydf wrote…</i></summary><blockquote>

```
mdl -i -w -s .md-style.rb toktok
toktok/integrations.md:39: MD013 Line length
toktok/integrations.md:66: MD013 Line length
```
</blockquote></details>

Done.

---

*[toktok/integrations.md, line 50 at r1](https://reviewable.io:443/reviews/toktok/website/81#-L3I83u-FYWS7hMfA71S:-L3IewqgULvASmZn_dOl:b-896fix) ([raw file](https://github.com/toktok/website/blob/0074f2f3fef5940ed065e80bcfdafadf53708056/toktok/integrations.md#L50)):*
<details><summary><i>Previously, robinlinden (Robin Lindén) wrote…</i></summary><blockquote>

Why not just "Java"? Since the other binding is marked as "Java/Scala."
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/81)*

**Comment by zoff99** [comment]
damn, Tox is everywhere.
Bash binding? helllooo :-)

File: toktok/integrations.md [hunk 1]
```diff
@@ -28,25 +34,43 @@ yourself to this page.
...
 *   [Go](https://github.com/josephyzhou/libtox) libtox
 *   [Go](https://github.com/kitech/go-toxcore) go-toxcore
 *   [Go](https://github.com/xhebox/gtox) gtox
 *   [Haskell](https://github.com/TokTok/hs-toxcore-c)
+*   [Java](https://github.com/zoff99/java_toxclient_example/tree/master/jni-c-toxcore)
 *   [Java/Scala](https://github.com/TokTok/jvm-toxcore-c)
 *   [JavaScript](https://github.com/TokTok/js-toxcore-c)
+*   [Julia](https://github.com/JuliaIO/Toxcore.jl)
+    (Attempt to make Toxcore accessible in Julia)
+*   [Node](https://github.com/saneki-discontinued/node-toxcore)
```
Comment (iphydf) [inline toktok/integrations.md]: This is js-toxcore-c in toktok, but an old discontinued and broken version. "Node == JS".

File: toktok/integrations.md [hunk 2]
```diff
@@ -28,25 +34,43 @@ yourself to this page.
...
 *   [Objective C](https://github.com/Antidote-for-Tox/objcTox)
+*   [Objective C](https://github.com/PotcFdk/ToxController)
```
Comment (iphydf) [inline toktok/integrations.md]: Add the names after, like we have with the Go bindings, so you can see it on the website.

File: toktok/integrations.md (hunk 1 above)
Comment (zoff99) [reply toktok/integrations.md]: removed

File: toktok/integrations.md (hunk 2 above)
Comment (zoff99) [reply toktok/integrations.md]: done

**Comment by SkyzohKey** [comment]
Can you use that PR to add the ones I've linked here ? https://github.com/TokTok/website/pull/79#pullrequestreview-90236128

Oh and there is also an electron front-end I did for fun, in others -> https://github.com/SkyzohKey/Tox-client

**Comment by iphydf** [comment]
:lgtm_strong:

---

Review status: 0 of 1 files reviewed at latest revision, 5 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/81#-:-L3T5LYeGWjgGFWPdH9Z:b-jbsf8z)*

**Pull request merged**